ridgeback_control/config/control.yaml

```diff
@@ -40,3 +40,21 @@ ridgeback_velocity_controller:
       max_velocity           : 2.0   # rad/s
       has_acceleration_limits: true
       max_acceleration       : 1.0   # rad/s^2
+
+ekf_localization:
+  frequency: 50
+  odom0: /ridgeback_velocity_controller/odom
+  odom0_config: [false, false, false,
+                 false, false, false,
+                 true, true, true,
+                 false, false, true]
+  odom0_differential: true
+  imu0: /imu/data
+  imu0_config: [false, false, false,
+                true, true, false,
+                false, false, false,
+                true, true, true]
+  imu0_differential: false
```

Comment:
What's the IMU frequency? If it's > 50 Hz, you may want to add the imu0_queue_size parameter and make it large enough to capture all the measurements.